docs/src/solvers/pinn.md

```diff
@@ -55,13 +55,19 @@ List of training strategies that are available now:
    to the spacing per direction.
  - `StochasticTraining(points)`: `points` number of sochastically sampled points from the domain. 
    In each optimization iteration, we randomly select a new subset of points from a full training set.
-- `QuasiRandomTraining(sampling_alg)`: The training set is generated on quasi-random low discrepency
-  sequences. See the [QuasiMonteCarlo.jl](https://github.com/SciML/QuasiMonteCarlo.jl) for the full
+- `QuasiRandomTraining(sampling_alg,points,minibatch)`: The training set is generated on quasi-random low discrepency
```

Comment:
I think minibatch should be a keyword argument. I actually don't quite get what it does from the description here. Is it the number of points per loss evaluation, where `points` is the total points from the quasi-random set, so the minibatch is the number of points to take in a subsample?

Reply:
`minibatch` is the number of subsets, where `points` is the number of points in minibatch. The number of the total points is `length(lb) * points * minibatch`, where `lb` is the lower bound and `length(lb)` is the dimensionality. 
On each iteration of training, it is randomly selected one of the minibatch.

```julia
using QuasiMonteCarlo
points = 3
lb = [0, 0]
ub = [1, 1]
sampling_alg = UniformSample()
minibatch = 4
s = QuasiMonteCarlo.generate_design_matrices(points,lb,ub,sampling_alg,minibatch)

julia> size(s)
(4,)
julia> size(s[1])
(2, 3)

```

Reply:
>minibatch is the number of subsets, where points is the number of points in minibatch. The number of the total points is length(lb) * points * minibatch, where lb is the lower bound and length(lb) is the dimensionality.
On each iteration of training, it is randomly selected one of the minibatch.

That's perfect. That should go in the docs.